### PR TITLE
Step2-display-grid-monitoring-for-integration-setting (Server)

### PR DESCRIPTION
**Pull request by Skyriixxx**

### CODE REVIEW

File: src/assets/server/rest/v1/schemas/setting/setting-grid-monitoring-set.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                      },
+                      "clientId": {
+                        "type": "string",
+                        "sanitize": "mongo"
+                      },
+                      "clientSecret": {
+                        "type": "string",
+                        "sanitize": "mongo"
+                      },
+                      "normalConsumption": {
```
Comment (LucasBrazi06) [inline src/assets/server/rest/v1/schemas/setting/setting-grid-monitoring-set.json:48]: I asked Liam to push the GridMonitoring settings (GridMonitoring.ts).
Check and align all levels.
You should have contacted Liam for knowing the setting interface!
image
Fix the naming in the front also.

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Check my comments

File: src/assets/server/rest/v1/schemas/setting/setting-grid-monitoring-set.json (hunk 1 above)
Comment (Skyriixxx) [reply src/assets/server/rest/v1/schemas/setting/setting-grid-monitoring-set.json]: Done

**Review by LucasBrazi06 — APPROVED** [review]
GUT

**Pull request merged**